index.html

```diff
@@ -53,215 +53,240 @@ <h5>LET'S CONNECT</h5>
-                    <li><a href="https://www.linkedin.com/in/giorgi-bokuchava-430252240/"><img
-                                src="Pictures/Linkedin.png" alt=""></a></li>
-                    <li><a href="https://angellist.com"><img src="Pictures/victory.png" alt=""></a></li>
-                    <li><a href="https://twitter.com"><img src="Pictures/twitter.png" alt=""></a></li>
-                    <li><a href="https://medium.com"><img src="Pictures/medium.png" alt=""></a></li>
+                    <li class="b1"><img src=Pictures/js.png>JavaScript</li>
+                    <li class="b2"><img src=Pictures/html.png>HTML</li>
+                    <li class="b3"><img src=Pictures/css.png>CSS</li>
                 </ul>
             </div>
-        </div>
-        <button class="but1">Get my resume</button>
-    </section>
-
-
 
+            <div class="frame">
+                <div class="frameworks">Frameworks</div>
+                <a href=""><img class="right" src="Pictures/right.png" alt=""></a>
+            </div>
 
+            <div class="sep">
+                <img src="Pictures/seperator.png" alt="">
+            </div>
 
+            <div class="skills">
+                <div class="skills1">Skills</div>
+                <a href=""><img class="right1" src="Pictures/right.png" alt=""></a>
+            </div>
 
+            <div class="sep">
+                <img src="Pictures/seperator.png" alt="">
+            </div>
 
 
-    <section class="last">
 
 
+        </section>
 
 
-        <div class="arrows">
-            <h7>Languages</h7>
-            <a href=""><img class="down" src="Pictures/down.png" alt=""></a>
-        </div>
 
 
-        <div class="js">
-            <ul>
-                <li class="b1"><img src=Pictures/js.png>JavaScript</li>
-                <li class="b2"><img src=Pictures/html.png>HTML</li>
-                <li class="b3"><img src=Pictures/css.png>CSS</li>
-            </ul>
-        </div>
 
-        <div class="frame">
-            <div class="frameworks">Frameworks</div>
-            <a href=""><img class="right" src="Pictures/right.png" alt=""></a>
-        </div>
 
-        <div class="sep">
-            <img src="Pictures/seperator.png" alt="">
-        </div>
 
-        <div class="skills">
-            <div class="skills1">Skills</div>
-            <a href=""><img class="right1" src="Pictures/right.png" alt=""></a>
-        </div>
 
-        <div class="sep">
-            <img src="Pictures/seperator.png" alt="">
-        </div>
+        <section class="contact">
+            <form action="https://formspree.io/f/xnqrdyay" method="POST">
+                <h4>Contact me</h4>
+                <p class="contact-para">If you have an application
+                    you are interested in developing, a feature that
+                    you need built or a project that
+                    needs coding. I’d love to help with it </p>
```

Comment:
- [ ] [OPTIONAL] From my point of view, I would suggest that you move this `<h4>` and `<p>` out of the form tag. This will look convenient and more understandable 😉 😉